packages/btrfs-progs/androidcompat.h.patch

```diff
@@ -0,0 +1,25 @@
+--- a/androidcompat.h
++++ b/androidcompat.h
+@@ -0,0 +1,22 @@
++#ifndef __ANDROID_H__
++#define __ANDROID_H__
++
++#ifdef __ANDROID__
++
++#define pthread_cancel(thread) pthread_kill((thread), SIGUSR1)
```

Comment:
Replacing `pthread_kill` in such way is *dangerous* since thread won't be terminated properly (if at all).

Sample issue related to `pthread_cancel`: https://github.com/termux/termux-packages/issues/4378

Reply:
This is copied from older upstream code. But yeah, they have dropped support for Android. So I won't package this.